### ISSUE TITLE
Update config to point logs to /dev/stdout

### ISSUE THREAD

**Issue by jbergknoff** [aptible/docker-nginx#37]
After PR #36, the configuration files don't point logs at `/dev/stdout` because old versions of docker were failing to write logs. Instead, the logs are directed to `/proc/self/fd/1`, which is where the `/dev/stdout` symlink leads. When legacy docker (< 0.12.0) is no longer in the picture, it may make sense to re-introduce the less obscure `/dev/stdout` for the sake of clarity/greppability/etc.

**Comment by fancyremarker** [comment]
Great, thanks for filing.

**Comment by travcunn** [comment]
It may be a good time to revisit this issue. This commit previously allowed `/dev/stdout` and `/dev/stderr` to work correctly: https://github.com/aptible/docker-nginx/commit/8e90f85d002c27f06f31f42fd74b456f6bf55b32

Here is an alternative way:

Dockerfile change:
```
RUN \
    ln -sf /dev/stdout /var/log/nginx/access.log \
    && ln -sf /dev/stderr /var/log/nginx/error.log
```

The Nginx config would contain these settings in the appropriate sections:
```
error_log  /var/log/nginx/error.log warn;
access_log  /var/log/nginx/access.log  main;
```

With this test, of course:
```
@test "It should log to STDOUT" {
  pkill nginx
  /usr/sbin/nginx > /tmp/nginx.log &
  curl localhost:80 > /dev/null 2>&1
  [[ -s /tmp/nginx.log ]]
}
```

I'm not sure how to trigger the error log in testing, but perhaps using an invalid Nginx config file would do the trick?

**Comment by fancyremarker** [comment]
@travcunn : I agree we should probably get rid of the obscure `/proc/self/fd/1` reference, especially now that (hopefully) not many folks are using Docker < 0.12.0. That said, the image does/should work as-is, so if you're not seeing log output, perhaps there's another unrelated issue? If you share the steps you took to launch your container (and the Docker version) such that you're not seeing access and error logs, I can investigate further.
